### ISSUE TITLE
need  help！！

### ISSUE THREAD

**Issue by liaohuyu** [danikula/AndroidVideoCache#156]
大神，不知道你看得懂中文不。网页翻译的可能不准，还是把中文附加上吧。
问题1：开始播放速度很慢，说是header的问题，略过后快了一点。
问题2：连接失败后没有自动重试。比如：我缓存了一部分，把网络断掉，它不会继续缓存了。
问题3：断点续传问题。我的url是可变的，但是文件却是不变的，怎么根据url和name一起缓存，而不是根据name缓存呢？
非常希望得到大神的帮助！万分感谢！
Great God, I don't know if you can read Chinese or not. The translation may be inaccurate, or append chinese.
Question 1: the playback speed is slow. It's header's problem, but it's a little faster after a while.
Problem 2: no auto retry after connection failure. For example: I cache a part and break the network. It won't continue caching.
Question 3: http problems. My URL is mutable, but the file is immutable. How do you cache it based on URL and name instead of the name cache?
Very much hope to get God's help! Thanks a million!

**Comment by danikula** [comment]
Question 3: read [README](https://github.com/danikula/AndroidVideoCache#providing-names-for-cached-files).

Question 1-2 provide as much more information as possible.

**Comment by liaohuyu** [comment]
问题1：从请求URL，到能播放的时候，时间间隔很久，大概8秒。略过header请求后，大概是1-3秒，但是，慢的时候依然很多。如果直接播放这个URL，1秒就能出来。
问题2，问题3，我再试试。感谢大神回答。
Question 1: from request URL to play, the time interval is long, about 8 seconds. After skipping the header request, it's about 1-3 seconds, but it's still a lot slower. If you play the URL directly, you can come out in 1 second.
Question 2, question 3, let me try again. Thank God for the answer.

**Comment by liaohuyu** [comment]
@danikula 
问题2：缓存中断后的处理。比如，我只缓存了一部分，然后由于某些原因缓存中断了，我播放到这里的时候想继续缓存，而不是直接失败，至少重新请求一次。现在怎么能过拿到缓存中断的回调呢？感谢大神回复！
Problem 2: handling after cache interrupts. For example, I only cache a portion of it, and then for some reason the cache is interrupted. I want to continue caching when I play here, rather than fail directly, at least once again. How can you get the callback of cache interrupt now? Thank God for your reply!
